The HawkEye Hook seems to be outdated

After getting a few errors while running HawkEye simultaneously with BlocksHub I decided to poke around a bit and figure out what was going on.

DataManager appears to have replaced the method of `DataManager#addEntry(BlockChangeEntry)` with `HawkEyeAPI#addEntry(BlockChangeEntry)` ([Link](https://github.com/bob7l/HawkReloaded/blob/master/src/uk/co/oliwali/HawkEye/util/HawkEyeAPI.java))

Both use static access, so theoretically it's just removing the DataManager import & reference with the HawkEyeAPI import & reference. I would make a PR, but this project doesn't work well with my IDE.

Here is the error;
```
[00:02:22] [Craft Scheduler Thread - 20/ERROR]: [WorldEdit] An unexpected error while handling a WorldEdit command
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
	at com.sk89q.worldedit.util.command.parametric.ParametricCallable.call(ParametricCallable.java:244) ~[FastAsyncWorldEdit-bukkit.jar:?]
	at com.sk89q.worldedit.util.command.SimpleDispatcher.call(SimpleDispatcher.java:125) ~[FastAsyncWorldEdit-bukkit.jar:?]
	at com.sk89q.worldedit.extension.platform.CommandManager$2.run(CommandManager.java:308) [FastAsyncWorldEdit-bukkit.jar:?]
	at com.boydti.fawe.object.FawePlayer$1.run(FawePlayer.java:115) [FastAsyncWorldEdit-bukkit.jar:?]
	at com.boydti.fawe.object.FawePlayer.queueAction(FawePlayer.java:143) [FastAsyncWorldEdit-bukkit.jar:?]
	at com.boydti.fawe.object.FawePlayer.runAction(FawePlayer.java:184) [FastAsyncWorldEdit-bukkit.jar:?]
	at com.sk89q.worldedit.extension.platform.CommandManager.handleCommandOnCurrentThread(CommandManager.java:297) [FastAsyncWorldEdit-bukkit.jar:?]
	at com.sk89q.worldedit.extension.platform.CommandManager$3.run(CommandManager.java:373) [FastAsyncWorldEdit-bukkit.jar:?]
	at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftTask.run(CraftTask.java:58) [patched_1.11.jar:git-Paper-976]
	at org.bukkit.craftbukkit.v1_11_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) [patched_1.11.jar:git-Paper-976]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) [patched_1.11.jar:git-Paper-976]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_111]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_111]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
Caused by: java.lang.NoSuchMethodError: uk.co.oliwali.HawkEye.database.DataManager.addEntry(Luk/co/oliwali/HawkEye/entry/DataEntry;)V
	at org.primesoft.blockshub.logger.bukkit.hawkEye.HawkEyeLogger.logBlock(HawkEyeLogger.java:109) ~[?:?]
	at org.primesoft.blockshub.Logic.logBlock(Logic.java:366) ~[?:?]
	at com.boydti.fawe.logging.LoggingChangeSet.add(LoggingChangeSet.java:56) ~[?:?]
	at com.boydti.fawe.object.HistoryExtent.setBlock(HistoryExtent.java:76) ~[?:?]
	at com.boydti.fawe.object.extent.ProcessedWEExtent.setBlock(ProcessedWEExtent.java:124) ~[?:?]
	at com.boydti.fawe.object.extent.ProcessedWEExtent.setBlock(ProcessedWEExtent.java:92) ~[?:?]
	at com.sk89q.worldedit.EditSession.setBlockFast(EditSession.java:1124) ~[?:?]
	at com.sk89q.worldedit.EditSession.setBlock(EditSession.java:1118) ~[?:?]
	at com.sk89q.worldedit.world.AbstractWorld.setBlock(AbstractWorld.java:83) ~[?:?]
	at com.sk89q.worldedit.function.block.BlockReplace.apply(BlockReplace.java:53) ~[?:?]
	at com.sk89q.worldedit.function.RegionMaskingFilter.apply(RegionMaskingFilter.java:53) ~[?:?]
	at com.sk89q.worldedit.function.visitor.RegionVisitor.resume(RegionVisitor.java:58) ~[?:?]
	at com.sk89q.worldedit.function.operation.Operations.completeBlindly(Operations.java:67) ~[?:?]
	at com.sk89q.worldedit.function.operation.Operations.completeSmart(Operations.java:75) ~[?:?]
	at com.sk89q.worldedit.EditSession.replaceBlocks(EditSession.java:1627) ~[?:?]
	at com.sk89q.worldedit.command.RegionCommands.replace(RegionCommands.java:289) ~[?:?]
	... 18 more
```

Hope this helps out and I'm hoping I don't come across as too "another developer nagging", just hoping to provide as much info as possible about a fix so it can hopefully be easy to do.

The WorldEdit setup involves Fast Async WorldEdit and WorldEdit. I do not believe they are related to this problem, however.

--- a/BlocksHub-HawkEye/src/org/primesoft/blockshub/logger/bukkit/hawkEye/HawkEyeLogger.java
+++ b/BlocksHub-HawkEye/src/org/primesoft/blockshub/logger/bukkit/hawkEye/HawkEyeLogger.java
@@ -53,7 +53,7 @@ import org.primesoft.blockshub.platform.bukkit.BukkitBaseEntity;
 import org.primesoft.blockshub.platform.bukkit.BukkitWorld;
 import uk.co.oliwali.HawkEye.DataType;
 import uk.co.oliwali.HawkEye.HawkEye;
-import uk.co.oliwali.HawkEye.database.DataManager;
+import uk.co.oliwali.HawkEye.util.HawkEyeAPI;
 import uk.co.oliwali.HawkEye.entry.BlockChangeEntry;
 import uk.co.oliwali.HawkEye.entry.BlockEntry;
 
@@ -106,11 +106,11 @@ public class HawkEyeLogger extends BukkitBaseEntity implements IBlockLogger {
         
         
         if (airNew) {
-            DataManager.addEntry(new BlockEntry(player.getName(), DataType.WORLDEDIT_BREAK, blockOld.getType(), blockOld.getData(), l));
+            HawkEyeAPI.addEntry(new BlockEntry(player.getName(), DataType.WORLDEDIT_BREAK, blockOld.getType(), blockOld.getData(), l));
             return;
         }
 
-        DataManager.addEntry(new BlockChangeEntry(player.getName(), DataType.WORLDEDIT_PLACE, 
+        HawkEyeAPI.addEntry(new BlockChangeEntry(player.getName(), DataType.WORLDEDIT_PLACE,
                 l,
                 blockOld.getType(), blockOld.getData(),
                 blockNew.getType(), blockNew.getData()));

